packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -84,7 +84,6 @@ class SingleChildScrollView extends StatelessWidget {
       physics: physics,
       viewportBuilder: (BuildContext context, ViewportOffset offset) {
         return new _SingleChildViewport(
-          key: key,
```

Comment:
Yeah, looks like a copy/paste error.  Harmless except if you pass in a global key.